plugin/templates/resource_name_fixed.mustache

```diff
@@ -16,12 +16,15 @@ package {{package}};
 
 import {{resourceNameGlobalPackageName}}.ResourceName;
 import {{resourceNameGlobalPackageName}}.ResourceNameType;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
 
 // AUTO-GENERATED DOCUMENTATION AND CLASS
 @javax.annotation.Generated("by GAPIC protoc plugin")
 public class {{className}} {{extensionKeyword}} {{parentInterface}} {
 
   private static final String FIXED_VALUE = "{{fixedValue}}";
+  private static final Map<String, String> valueMap = ImmutableMap.of("FIXED_VALUE", FIXED_VALUE);
```

Comment:
Why not have it keyed with an empty string, like the untyped case?

Reply:
Now keying on empty string; adjusted method comments to reflect that.